openlibrary/templates/book_providers/openstax_download_options.html

```diff
@@ -0,0 +1,11 @@
+$def with(openstax_id)
+
+$ url = 'https://openstax.org/details/books/' + openstax_id
+
+<hr>
+<div class="cta-section">
+  <p class="cta-section-title">$_('Download Options')</p>
+    <ul class="ebook-download-options">
+      <li><a href="$url" title="$_('Download book from OpenStax')">$_('More at OpenStax')</a></li>
```

Comment:
They're all available as PDFs, so let's advertise that.

```suggestion
      <li><a href="$url" title="$_('Download PDF from OpenStax')">$_('PDF')</a></li>
      <li><a href="$url">$_('More at OpenStax')</a></li>
```